packaging/src/docker/conf/hive-site.xml

```diff
@@ -44,6 +44,10 @@
         <name>hive.exec.submit.local.task.via.child</name>
         <value>false</value>
     </property>
+    <property>
+        <name>hive.compactor.worker.threads</name>
+        <value>1</value>
+    </property>
```

Comment:
This configuration is required to run major compactions. I created this patch while while I was debugging `OPTIMIZE TABLE ... REWRITE DATA` of Iceberg tables.

```
2025-02-09T13:03:22,303  WARN [main] server.HiveServer2: Invalid number of Compactor Worker threads(0) on HS2
2025-02-09T13:03:22,315  WARN [main] server.HiveServer2: No default compaction pool configured, all non-labeled compaction requests will remain unprocessed!
```